src/java.base/share/classes/sun/security/x509/AVA.java

```diff
@@ -640,7 +640,7 @@ private String toKeyword(int format, Map<String, String> oidMap) {
      */
     public String toString() {
         return toKeywordValueString
-            (toKeyword(DEFAULT, Collections.emptyMap()));
+            (toKeyword(DEFAULT, Collections.emptyMap()), Boolean.TRUE);
```

Comment:
I don't think you need to use `Boolean` here, you can just use the primitive boolean data type as the last parameter, and pass `true`.